salesagility/SuiteCRM
# Feature/google calendar admin

Move Google Calendar Integration from System Settings to Administration section.
 
## Description

The current google calendar authentication is located within the System Settings. This pull request will take the google calendar authentication from the System Settings section and put it within its own Administration section (Google Calendar).

## Motivation and Context
This change is to allow the feature to be located more easily for the user

## How To Test This

1. Go to: Admin section.
2. Locate "Google Calendar" Section
3. Click "Authentication" Link
4. Select Google Calendar Authentication JSON File
5. Click Save
6. Re-enter "Authentication" link Configured should be Green

7. Google Calendar integration should work as expected

## Types of changes


### Final checklist
